apps/api/src/app/workflows-v2/usecases/build-step-issues/build-step-issues.usecase.ts

```diff
@@ -311,10 +312,25 @@ export class BuildStepIssuesUsecase {
     }
   }
 
-  private mapAjvErrorToMessage(error: ErrorObject<string, Record<string, unknown>, unknown>): string {
+  private mapAjvErrorToMessage(
+    error: ErrorObject<string, Record<string, unknown>, unknown>,
+    stepType: StepTypeEnum
+  ): string {
+    if (stepType === StepTypeEnum.IN_APP) {
+      if (error.keyword === 'required') {
+        return 'Subject or body is required';
+      }
+      if (error.keyword === 'minLength') {
+        return `${capitalize(error.instancePath.replace('/', ''))} is required`;
+      }
+    }
+
     if (error.keyword === 'required') {
       return `${capitalize(error.params.missingProperty)} is required`;
     }
+    if (error.keyword === 'minLength') {
+      return `${capitalize(error.instancePath.replace('/', ''))} is required`;
+    }
```

Comment:
what are the message error in both of those errors?

just to clarify, the second one is boyscouting and is not related to this PR scope, right?